runtime/js/11_workers.js

```diff
@@ -269,7 +240,7 @@
               throw new Error("Unhandled error event reached main worker.");
             } else {
               core.opSync(
-                "op_host_unhandled_error",
+                "op_worker_unhandled_error",
```

Comment:
Nitpick: this op is called from the "host" side, so I'd prefer to keep `op_host_` prefix